packs/data/equipment.db/lawbringers-lasso.json

```diff
@@ -35,7 +35,30 @@
             }
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "action:grapple",
+                    "target:trait:chaotic"
+                ],
```

Comment:
Since this only applies to grapple checks with the lasso we should probably make a toggle.